metacoadd/exposure.py

```diff
@@ -255,40 +197,6 @@ def _set_meta(self):
         }
 
 
-class ExpList(list):
```

Comment:
Why removing the `ExpList` class?

Reply:
You're right, it should be in another PR.
Main argument : it adds complexity to the code without much benefit (just check the type of elements when appending elements). That's better left to type checking (either dynamically with an `all(isintance(elem, Exposure) for elem in explist)` at the beginning of the code using it, or static type checking.

Reply:
Ok, in order to separate things, I reintroduced the ExpList class in this PR.